app/domain/rotation/rotators/example/example_rotator.rb

```diff
@@ -0,0 +1,32 @@
+module Rotation
+  module Rotators
+    module Example
+
+      class ExampleRotator
+
+        def initialize
+          @cnt = 0
+        end
+
+        def required_variables(annotated_variable)
```

Comment:
Actually, looks more like it would return:
```yml
%w(region access_key_id secret_access_key secret_key_proxy)
```

Reply:
```
%w(region access_key_id secret_access_key secret_key_proxy)
```
is correct.  

It's a way for the rotator to say what information it needs to perform it's calculation.  Similarly, the `new_values` method allows the rotator to specify that multiple variables need to be updated.  This will all be spelled out clearly in the docs.